charts/artifact-hub/templates/hub_deployment.yaml

```diff
@@ -71,6 +71,10 @@ spec:
               protocol: TCP
           resources:
             {{- toYaml .Values.hub.deploy.resources | nindent 12 }}
+          livenessProbe:
```

Comment:
Can you update the [`values.schema.json` file](https://github.com/artifacthub/hub/blob/062df42b9e294b02f960e726705d38ab882d1db8/charts/artifact-hub/values.schema.json#L187-L223) with these new entries? In addition to adding a block for each, they both should be marked as required with a default of `{}`. Please don't forget setting the corresponding reference to the kubernetes json schema to make them look nice in the [values schema viewer](https://artifacthub.io/packages/helm/artifact-hub/artifact-hub?modal=values-schema&path=hub.deploy.resources).